CHANGELOG.md

```diff
@@ -0,0 +1,215 @@
+
+-   Fix: Regression introduced in 2.3 which resulting in requesting all
+    runs for a job, even if most were cached
+    [JENKINS-40087](https://issues.jenkins-ci.org/browse/JENKINS-40087)
+
+#### 2.3 (Nov 22, 2016)
+
+-   Fix: Stage View showed multiple rows if displayName changed for a
+    run (community PR, thank you!) -
+    [JENKINS-34538](https://issues.jenkins-ci.org/browse/JENKINS-34538)
+
+#### 2.2 (Nov 2, 2016)
+
+-   CSS Fix: excess whitespace on sides of stages table with very high
+    resolution monitors
+
+#### 2.1 (Sept 30, 2016)
+
+-   Fix: Prevent errors in deeply nested blocks from overflowing the
+    stage status window
+    ([JENKINS-37945](https://issues.jenkins-ci.org/browse/JENKINS-37945))
+    -   Also filters out block start/end steps from that and log display
+        (reverts to 1.7 behavior)
+-   Fix: hide NotExecuted steps from log display (user is resuming from
+    checkpoint, these steps never ran on this run)
+-   CSS Fix: prevent excessively long error messages from overflowing
+    stage status popup
+-   Feature: display timing information for steps in log view
+-   Don't show log-expand icons if there is no log message for a step
+    ([JENKINS-31833](https://issues.jenkins-ci.org/browse/JENKINS-31833))
+
+#### 2.0 (Aug 30, 2016)
+
+-   Feature: support visualization of block-scoped stages (labelled
+    blocks)
+    ([JENKINS-26107](https://issues.jenkins-ci.org/browse/JENKINS-26107))
+-   **Major feature**: complete rewrite of the flow analysis algorithms
+    ([JENKINS-34038](https://issues.jenkins-ci.org/browse/JENKINS-34038))
+    -   Needed to support block-scoped stage, and originally intended to
+        allow for visualizing parallels
+    -   Scalability: simpler and more efficient APIs allow for analyzing
+        flows in a logical one-pass fashion.
+    -   Performance: allows for far greater performance.  Currently
+        roughly at parity with 1.7, but
+        once [JENKINS-37569](https://issues.jenkins-ci.org/browse/JENKINS-37569) is
+        done, should be \*much\* faster.
+    -   Optimization: includes one final round of caching improvements.
+         We can now serve requests about individual stages/nodes from
+        the cache, and caches use less memory.
+
+#### 1.7 (Aug 6, 2016)
+
+-   Performance/bugfix: massively improve serverside run/stage caching,
+    by greatly limiting excess cache invalidation
+    -   Cache no longer fully invalidates on renaming a pipeline job:
+        instead the old cache entries are removed and new ones created
+        with the new job name
+    -   Bugfix: Need to invalidate run cache results when their job is
+        deleted
+        ([JENKINS-36865](https://issues.jenkins-ci.org/browse/JENKINS-36865))
+    -   Known issue: run cache can no longer serve requests for data
+        about individual stages/nodes. 
+        -   Mitigation: UI switches to use API below, so it just gets
+            this data with the run request
+
+&nbsp;
+
```

Comment:
Looks like a leftover glitch from some funky formatting on [the wiki page](https://wiki.jenkins.io/display/JENKINS/Pipeline+Stage+View+Plugin
). Could probably wipe out this whole block and bring the two sets of bullets into one list.

Pic of the old wiki page:

![image](https://user-images.githubusercontent.com/21689198/70249127-f3e2cd00-1749-11ea-9e8c-e508c5fea2d4.png)